Config/common/countries.cwt

```diff
@@ -32,7 +32,7 @@ enums = {
 		path = "game/common/country_tags"
 		start_from_root = yes
 		name = {
-			enum_name = scalar
+			enum_name = "scalar"
```

Comment:
Intended, otherwise it will pick up `dynamic_tags  = yes` in zz_dynamic_countries.txt